loops-and-arrays.js

```diff
@@ -30,6 +33,18 @@ const numbers = [2, 4, 5, 29, 38];
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
+console.log("Original array:", numbers);
+
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] === 2 || numbers[i] === 4 || numbers[i] === 38) {
```

Comment:
Je hebt nu letterlijk de getallen uit de array gehaald. Maar stel je voor de array heeft 2000 entries met getallen, ga je die dan allemaal benoemen in de if statement? 

Dat lijkt mij niet! Je wilt dus eigenlijk gebruik gaan maken van het modulo karakter -> %. Verdiep je daar nog even in, het is het waard!